packages/app-project/stores/Project.js

```diff
@@ -41,7 +41,7 @@ const Project = types
     get defaultWorkflow() {
       const activeWorkflows = self.links['active_workflows']
       let singleActiveWorkflow
-      if (activeWorkflows.length === 1) {
+      if (activeWorkflows?.length === 1) {
```

Comment:
Is there a change that this would be undefined?

Reply:
It came up when I was first testing this branch but I don’t remember how.

Reply:
It seems really unlikely that Panoptes would not serialize this back to us, so likely has to do with how the project resource is being mocked for tests. Maybe we can setup rosie in the project app to consistent mocks for testing?

Reply:
It’s undefined for projects that don’t exist in Panoptes , but I don’t remember how I triggered that.

Reply:
Whatever that error was, I can't reproduce it either in the dev site or in the tests. I'm going to take this existence check out.